src/main/java/me/zeroeightsix/kami/mixin/client/MixinMinecraft.java

```diff
@@ -33,6 +40,24 @@
     @Shadow public GuiIngame ingameGUI;
     @Shadow public boolean skipRenderWorld;
     @Shadow public SoundHandler soundHandler;
+    @Shadow public RayTraceResult objectMouseOver;
+    @Shadow public PlayerControllerMP playerController;
+    @Shadow public EntityRenderer entityRenderer;
+
+    @Inject(method = "rightClickMouse", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/entity/EntityPlayerSP;getHeldItem(Lnet/minecraft/util/EnumHand;)Lnet/minecraft/item/ItemStack;"), cancellable = true)
+    public void processRightClickBlock(CallbackInfo ci) {
+        if (CrystalAura.INSTANCE.isActive()) {
+            ci.cancel();
+            for (EnumHand enumhand : EnumHand.values()) {
```

Comment:
can't you just use `EnumHand.forEach`?

Reply:
Minecraft code moment, and it is just some code style lol

Reply:
yea, it technically compiles to a forEach anyways, it's just cleaner to use forEach imo

Reply:
yea, it technically compiles to a forEach anyways, it's just cleaner to use forEach imo

Reply:
Java Array doesn't have `forEach` lol. And enhanced for loop is already clean enough imo

Reply:
cringe java array

Reply:
♿